samples/bot-message-reminder/nodejs/Readme.md

```diff
@@ -0,0 +1,82 @@
+## Message reminder with messaging extension action
+
+This sample shows feature where user can schedule a task from messaging extension action and get a reminder card at scheduled time.
```

Comment:
This sample shows a feature where user can schedule a task from messaging extension action and get a reminder card at a scheduled time.